girder_worker/__init__.py

```diff
@@ -33,20 +31,17 @@
 logger = log_utils.setupLogger(config)
 
 
-@add_metaclass(abc.ABCMeta)
 class GirderWorkerPluginABC(object):
```

Comment:
Should this still be named `ABC` if it's not a true ABC?  Seems like just a regular base class now.

Reply:
I don't think it should, however I'm not trying to make a breaking change that would be trouble for all downstream plugins. As written, this should be non-breaking.